README.md

```diff
@@ -85,6 +85,59 @@ git fetch upstream-code main
 4. Fix the conflicts or other errors. **Note**, that`./rebase.sh` script also apllies the patches from the [`.rebase`](https://github.com/che-incubator/che-code/tree/main/.rebase) directory. Sometimes, it also requires some updates there.
 5. Open a PR with your changes.
 
+## Whitelabel/branding
+ 
+There is an ability to apply branding for some UI elements. The original data can be overridden by execution [branding.sh script](https://github.com/che-incubator/che-code/blob/main/branding/branding.sh). The script expects that the branding resources are placed in the [branding folder](https://github.com/che-incubator/che-code/tree/main/branding). 
+
+The crucial one is: `product.json` file. 
```

Comment:
```suggestion
The `branding/product.json` file is crucial.
```

Reply:
thank you, applied manually as there was a conflict